drivers/flash/flash_stm32h7x.c

```diff
@@ -467,7 +468,8 @@ static struct flash_stm32_sector_t get_sector(const struct device *dev, off_t of
 
 	bool bank_swap;
 	/* Check whether bank1/2 are swapped */
-	bank_swap = (READ_BIT(FLASH->OPTCR, FLASH_OPTCR_SWAP_BANK) == FLASH_OPTCR_SWAP_BANK);
+	bank_swap = stm32_reg_read_bits(&FLASH->OPTCR, FLASH_OPTCR_SWAP_BANK) ==
+		    FLASH_OPTCR_SWAP_BANK;
```

Comment:
Proposal to fix the missing empty line between variable definition and the instructions that follows:
```suggestion
	/* Check whether bank1/2 are swapped */
	bool bank_swap = stm32_reg_read_bits(&FLASH->OPTCR, FLASH_OPTCR_SWAP_BANK) ==
			 FLASH_OPTCR_SWAP_BANK;
```
feel from to discard this review comment.